config/environments/production.rb

```diff
@@ -89,3 +89,5 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 end
+
+config.force_ssl = true
```

Comment:
Cách thức truyền đi của SSL như sau:
1. Server gửi một bản sao của public key của nó đến cho máy khách
2.Trình duyệt tạo ra một sessionkey và mã hóa nó với publickey bất đối xứng của máy chủ rồi gửi đến cho máy chủ ( bất đối xứng là bất cứ ai cũng có thể dùng publickey để mã hóa nhưng để giải mã chỉ có người nào có privatekey)
3. Server giải mã sessionkey và tạo ra một session key với dạng mã hóa đối xứng. ( mã hóa đối xứng là chỉ có 1 key vừa mã hóa và vừa giải mã thông điệp)
4. Lúc này server và trình duyệt thưc hiện các mã hóa và giải mã tất cả các dữ liệu được truyền với session key. Điều này cho phép một kênh an toàn vì chỉ trình duyệt và máy chủ biết session key, và session key chỉ được sử dụng cho phiên truy cập đó. Nếu trình duyệt là để kết nối đến máy chủ cùng vào ngày hôm sau, một session key mới sẽ được tạo ra.